src/responder/common/data_provider/rdp_message.c

```diff
@@ -269,6 +269,10 @@ static void rdp_message_send_and_reply_done(DBusPendingCall *pending,
     sbus_req = talloc_get_type(ptr, struct sbus_request);
 
     ret = rdp_process_pending_call(sbus_req, pending, &reply);
+    if (ret != EOK) {
+        /* Something bad happened. Just kill the request. */
+        goto done;
+    }
     if (reply == NULL) {
```

Comment:
This change is not really required because `reply` is initialized to `NULL` in the function `rdp_process_pending_call`
The question is whether we need a `if (reply == NULL)` branch after this change.

I like consistency but It would be good to know why @pbrezina  wrote is in such way,
BTW return value of `rdp_process_pending_call` is checked on line 203 but here the output argument was checked.